backend/src/db/payments.js

```diff
@@ -0,0 +1,26 @@
+const models = require('../models');
+const db = require('../sequalize_pg');
+
+const createPayment = async (paymentObj) => {
+    await db.sync();
+    models.payments.create(paymentObj);
```

Comment:
Should this line be returned so that it can be waited in the caller's side correctly?